+precice/@SolverInterface/private/preciceGateway.cpp

```diff
@@ -313,7 +315,24 @@ class MexFunction: public matlab::mex::Function {
                 interface->setMeshQuadWithEdges(meshID[0],firstVertexID[0],secondVertexID[0],thirdVertexID[0],fourthVertexID[0]);
                 break;
             }
-            
+            case FunctionID::setMeshAccessRegion:
+            {
+                const TypedArray<int32_t> meshID = inputs[1];
+                const TypedArray<double> boundingBox = inputs[2];
+                interface->setMeshAccessRegion(meshID[0],&*boundingBox.begin());
+                break;
+            }
+            case FunctionID::getMeshVerticesAndIDs:
+            {
+                const TypedArray<int32_t> meshID = inputs[1];
+                const TypedArray<int32_t> size = inputs[2];
+                const TypedArray<int32_t> ids = inputs[3];
+                const TypedArray<double> positions = inputs[4];
+                interface->getMeshVertices(meshID[0],size[0],&*ids.begin(),&*positions.begin());
+                outputs[0] = factory.createArrayFromBuffer<double>({1,size[0]}, std::move(positions));
+                outputs[1] = factory.createArrayFromBuffer<int32_t>({1,size[0]}, std::move(ids));
```

Comment:
Primarily this seems correct. There is no way to test right now so our best bet is to as close to other function methodologies as possible. For example in `setMeshVertices` the coordinates are inputs and vertex IDs are outputs. The output is generated and returned in the following way:
https://github.com/precice/matlab-bindings/blob/dd9f642b45614015aa899f24ea8b0ac6d0cca854/%2Bprecice/%40SolverInterface/private/preciceGateway.cpp#L241-L244
try to follow the same methodology here.